### PR TITLE
Add blog for qiskit trebugger (new format)

### PR DESCRIPTION
**Pull request by TheGupta2012**
In continuation of [PR 331](https://github.com/unitaryfund/unitary.fund/pull/331).
Add qiskit trebugger blog with new CLI view.

### CODE REVIEW

File: posts/_markdown/2023_qiskit_trebugger.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Timeline Debugger for the Qiskit Transpiler
+date: 31 07 2023
```
Comment (nathanshammah) [inline posts/_markdown/2023_qiskit_trebugger.md:3]:
```suggestion
day: 31
month: 07 
year: 2023
```

File: posts/_markdown/2023_qiskit_trebugger.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+**Qiskit Trebugger (transpiler-debugger)** is a tool which provides a visual representation of the transpilation process. It was developed as an interactive tool with multiple views catering to the needs of different users. Available as a [python package on PyPI](https://pypi.org/project/qiskit-trebugger/), it can be used as a jupyter widget or  a lightweight CLI tool to understand how a circuit goes through the qiskit transpiler. The following image shows a preview of the CLI view - 
+
+<img src="../../images/2023_qiskit_trebugger_cli_full.png">
```
Comment (nathanshammah) [inline posts/_markdown/2023_qiskit_trebugger.md:14]:
```suggestion
```

**Comment by nathanshammah** [comment]
#361 is merged! Thanks for your patience @TheGupta2012 and congrats for the great post!

**Comment by TheGupta2012** [comment]
Thanks for reviewing again @nathanshammah !

**Pull request closed without merging**